migrations/sql/V2021.04.09.1500__SubnationalDemography.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE "subnational_region" (
+"id" SERIAL NOT NULL ,
+"country" INTEGER NOT NULL ,
+"name" TEXT NOT NULL ,
+PRIMARY KEY ("id") ,
+UNIQUE ("id", "name")
+);
+
+ALTER TABLE "subnational_region" ADD FOREIGN KEY ("country") REFERENCES "country" ("nid");
+
+CREATE TABLE "subnational_demographic_statistic" (
+"region" INTEGER NOT NULL ,
+"age_from" INTEGER NOT NULL ,
+"age_to" INTEGER NOT NULL ,
+"value" DECIMAL NOT NULL ,
+"year" INTEGER NOT NULL ,
```

Comment:
if this is going to be large, do you want to chuck indexes onto these at all? it will typically be fastest to do that after construction of a very large table though

Reply:
I'll come back to this one after the first round of data is done...